apps/web/src/components/Publication/OpenAction/UnknownModule/Decent/DecentAction.tsx

```diff
@@ -0,0 +1,155 @@
+import type { Amount } from '@hey/lens';
+import type { UIData } from 'nft-openaction-kit';
+import type { FC } from 'react';
+
+import LoginButton from '@components/Shared/Navbar/LoginButton';
+import MetaDetails from '@components/Shared/Staff/MetaDetails';
```

Comment:
if it's reused maybe move this file to shared folder?

Reply:
It's used in one place only 👍